src/environments/status-descriptions.ts

```diff
@@ -0,0 +1,43 @@
+import { LearningObject } from '@entity';
+import { CollectionService } from 'app/core/collection.service';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class StatusDescriptions {
+
+  private templates = {
+    unreleased() {
+      return 'This learning object is visible only to you. Submit it for review to make it publicly available.';
+    },
+    rejected() {
+      return 'This learning object was rejected. Contact your review team for further information.';
+    },
+    waiting(collection: string) {
+      return `This learning object is waiting to be reviewed by the ${collection} review team. It is not yet released and cannot be edited until the review process is complete.`;
+    },
+    review(collection: string) {
+      return `This learning object is currently under review by the ${collection} review team. It is not yet released and cannot be edited until the review process is complete.`
+    },
+    proofing() {
+      return 'This learning object is currently undergoing proofing by the CLARK editorial team. It is not yet released and cannot be edited until the proofing process is complete.'
+    },
+    released(collection: string) {
+      return `This learning object is released in the ${collection} collection and can be browsed for.`;
+    }
+  };
+
+  constructor(private collectionService: CollectionService) { }
+
+  async getDescription(status: LearningObject.Status, collectionAbbreviation?: string) {
+    const needsCollection = [LearningObject.Status.RELEASED, LearningObject.Status.WAITING, LearningObject.Status.REVIEW];
```

Comment:
I don't follow the `needsCollection` variable being a set of statuses? Am I missing some context, or is there a better name we could use here?

Reply:
The `needsCollection` variable here dictates whether or not the status uses the name of a collection in its copy and 'needs" to fetch the full 'collection' from the CollectionService given its abbreviated name.

Reply:
Ah that makes sense. Maybe we could add JSDoc to the function that says just that. It's pretty clear now, but I did not pick that up at first

Reply:
Ah that makes sense. Maybe we could add JSDoc to the function that says just that. It's pretty clear now, but I did not pick that up at first

Reply:
Can do